tests/contrib/aiopg/test.py

```diff
@@ -211,6 +211,32 @@ def test_user_specified_service(self):
         assert len(spans) == 1
         assert spans[0].service != "mysvc"
 
+    @mark_asyncio
+    @run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
+    def test_trace_span_name_v0_schema(self):
+        conn = yield from aiopg.connect(**POSTGRES_CONFIG)
+        Pin.get_from(conn).clone(tracer=self.tracer).onto(conn)
+        yield from (yield from conn.cursor()).execute("select 'blah'")
+        conn.close()
+
+        spans = self.get_spans()
+        assert spans, spans
+        assert len(spans) == 1
+        assert spans[0].name == "postgres.query"
+
+    @mark_asyncio
+    @run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v1"))
+    def test_trace_span_name_v1_schema(self):
+        conn = yield from aiopg.connect(**POSTGRES_CONFIG)
+        Pin.get_from(conn).clone(tracer=self.tracer).onto(conn)
+        yield from (yield from conn.cursor()).execute("select 'blah'")
+        conn.close()
+
+        spans = self.get_spans()
+        assert spans, spans
+        assert len(spans) == 1
+        assert spans[0].name == "postgres.query"
```

Comment:
it should be `postgresql.query`

Reply:
I'm going to fix this as part of https://github.com/DataDog/dd-trace-py/pull/5632 which deals with service naming (this PR is dealing with operation name)

Reply:
Aren't we talking about an operation name here?

Reply:
yes, my bad!   I will update all the psql tests/code